signup/tool/src/bundle/messages_es.properties

```diff
@@ -830,16 +829,6 @@ tab_event_action=Acci\u00f3n
 
 tab_event_availability=Estado
 
-tab_event_signed=Inscritos
-
-tab_event_total_attendees=Participantes (Total\: {0})
-
-tab_event_total_waiting_list=Lista de espera (Total\: {0})
-
-tab_event_timeslot_attendees=Participantes: {0}
-
-tab_event_timeslot_waiting_list=En espera: {0}
```

Comment:
I'm not sure about deleting this block...

Reply:
I think the properties were removed or renamed from the EN translation in master, that's why they are not present anymore.